packages/client/wallets/web3auth-adapter/src/auth.ts

```diff
@@ -11,10 +6,6 @@ export const parseToken = (token: any) => {
             typeof window !== "undefined" ? window.atob(base64) : Buffer.from(base64, "base64").toString();
         return JSON.parse(jsonPayload || "");
     } catch (err) {
-        logError("[PARSE_TOKEN] - ERROR", {
```

Comment:
why was this dropped

Reply:
Two reasons:
- Doesn't seem that useful.
- I didn't want to copy over our entire logging setup.